version 1.2 (update)

Nickesuw opened the pull request.

TamtamHero committed:
clean deprecated WIDE keyword

Commit:
Bump version, remove WebUSB URL

Commit:
Merge branch 'prepare-1.6' of AppDave/ledger-app-waves into master

Commit:
Merged PR #8 from wavesplatform

TamtamHero committed:
Update os_global_pin_is_validated check

TamtamHero committed:
Bump version 1.1.1

TamtamHero committed:
Fix os_global_pin_is_validated check on Nano X

TamtamHero committed:
Bump version 1.1.2

Commit:
updating gitignore

Commit:
updating twisted edwards curve type

Commit:
add missing includes

Commit:
bump version to 1.1.3

Commit:
fix CURVE_HEADER macro field order

Commit:
fix: build for lns+

Jdumas-Ledger commented:
Hello, 
Would it be possible to merge this PR on LedgerHQ ? :) We don't use to review the PR in your repository.
Btw, it would be easier to get in contact ? Do you have any discord or telegram ? 
Here is mine : Jean - Ledger#8507
See you

Commit:
src: Update glyphs.h includes so that they don't use relatives includes

This is necessary as this file is automatically generated and it's generation location is going to change

(cherry picked from commit 0e156684f453381a035d0462b93baf658235f4c9)

xchapron-ledger committed:
Merge pull request #12 from LedgerHQ/adapt-to-new-sdk

Merge P1 deployed changes on master

xchapron-ledger committed:
Merge pull request #13 from LedgerHQ/xch/new-sdk-fix

src: Update glyphs.h includes so that they don't use relatives includes

nroggeman-ledger committed:
Do not use cx_eddsa_get_public_key_internal for a function existing in SDK

tdejoigny-ledger committed:
Merge pull request #15 from LedgerHQ/rename-local-function

Do not use cx_eddsa_get_public_key_internal for a function existing in SDK
